README_EN.md

```diff
@@ -0,0 +1,141 @@
+| **data**                    | Data dell’informazione              | Date of notification            | YYYY-MM-DD HH:MM:SS Italian time (ISO 8601)           | 2020-03-05 12:15:45 |                   |
+| **stato**                   | Stato di riferimento                | Country of reference            | ISO 3166-1 alpha-3 | ITA                  |
+| **codice_regione**          | Codice della Regione (ISTAT 2019)   | Code of the Region (ISTAT 2019) | Number             | 13                   |
+| **denominazione_regione**   | Denominazione della Regione         | Name of the Region              | String              | Abruzzo              |
+| **codice_provincia**        | Codice della Provincia (ISTAT 2019) | Code of the Province            | Number             | 067                  |
+| **denominazione_provincia** | Denominazione della provincia       | Name of the Province            | String              | Teramo               |
+| **sigla_provincia**         | Sigla della Provincia               | Province abbreviation           | String              | TE                   |
+| **lat**                     | Latitudine                          | Latitude                        | WGS84              | 42.6589177           |
+| **long**                    | Longitudine                         | Longitude                       | WGS84              | 13.70439971          |
+| **totale_casi**             | Totale casi positivi                | Total amount of positive cases  | Number             | 3                    |
+
+*The Autonomous provinces of Trento and Bolzano are listed separately in the "region name" but they both share the code 04 of Trentino Alto Adige as "codice_regione".* 
+
+*Each Region has a "fake" province named `In fase di definizione/aggiornamento` (in the definition/updating phase) with province code from 979 to 999. This code is used to indicate that the data was not yet assigned to a specific province.*
+
+*A JSON file containing all the information for all the dates is made available in at the path: `dati-json/dpc-covid19-ita-province.json`*
+
+### National trend
+
+**Directory:** `dati-andamento-nazionale`
+
+**Path daily file:** `dpc-covid19-ita-andamento-nazionale-yyyymmdd.csv` (example: `dpc-covid19-ita-andamento-nazionale-20200224.csv`)
+
+**Recap file:** `dpc-covid19-ita-andamento-nazionale.csv`
+
+
+| Name               | Descrizione  (Italian)                      | Description                            | Format            | Example              |
+|----------------------------|-----------------------------------|----------------------------------------|--------------------|----------------------|
+| **data**                       | Data dell’informazione            | Date of notification                   | YYYY-MM-DD HH:MM:SS Italian time (ISO 8601)           | 2020-03-05 12:15:45 |                                  |                                        |                    |                      |
```

Comment:
```suggestion
| **data**                       | Data dell’informazione            | Timestamp                   | YYYY-MM-DD HH:MM:SS Italian time (ISO 8601)           | 2020-03-05 12:15:45 |                                  |                                        |                    |                      |
```

Reply:
"Italian time" -> CET / UTC+1 ?

Reply:
@brucellino vero meglio essere precisi.
Mi chiedo pero' cosa succede col passaggio all'ora legale che formalmente e' CEST (UTC+2).  Non so come si possa indicare. 
Forse mettere CET e mettere una nota dicendo "from March 29th all timestamps are to be considered as CEST".

Detto questo forse non e' un gran problema perche' l'orario e' solo relativo all'update, non ha particolare rilevanza in questo caso.

Reply:
"Europe/Rome"?